libs/mrwebrtc-unityplugin/src/api.cpp

```diff
@@ -87,8 +87,16 @@ mrsNativeRenderer_EnableRemoteVideo(mrsNativeVideoHandle nativeVideoHandle,
                                     int textureCount) noexcept {
   UnityLogger::LogDebug("mrsNativeRenderer_EnableRemoteVideo");
   NativeRenderer* nativeVideo = static_cast<NativeRenderer*>(nativeVideoHandle);
-  //NativeRenderer* nativeVideo = (NativeRenderer*)nativeVideoHandle;
-  nativeVideo->EnableRemoteVideo(format, textures, textureCount);
+  nativeVideo->SetRemoteVideoTextures(format, textures, textureCount);
```

Comment:
This can throw due to `resize()`. Either it should be caught inside the method, and the method marked as `noexcept`, or caught here, to avoid propagating exceptions and for correctness since this function is marked as `noexcept` itself.